src/lib.rs

```diff
@@ -664,6 +664,24 @@ impl<'a, T: Clone + Num + Neg<Output = T>> Neg for &'a Complex<T> {
     }
 }
 
+impl<T: Clone + Num + Inv<Output = T>> Inv for Complex<T> {
+    type Output = Complex<T>;
+
+    #[inline]
+    fn inv(self) -> Complex<T> {
+        self.inv()
```

Comment:
I'm not sure, but you might have to make this explicitly `(&self).inv()` to avoid recursion.